src/app/lib/config/services/arabic.js

```diff
@@ -63,6 +63,20 @@ export const service = {
     },
     showAdPlaceholder: true,
     showRelatedTopics: true,
+    podcastPromo: {
+      title: 'البودكاست',
+      brandTitle: 'بي بي سي إكسترا',
+      brandDescription:
+        'بي بي سي إكسترا بودكاست يناقش كل أمور حياتنا اليومية، يأتيكم ثلاثة أيام أسبوعياً',
+      image: {
+        src: 'https://ichef.bbci.co.uk/images/ic/3000x3000/p09kh14x.jpg',
```

Comment:
I think this needs to be `$recipe` instead of `3000x3000` so that the `srcSet` attribute is generated correctly as shown here https://github.com/bbc/simorgh/blob/latest/src/app/containers/PodcastPromo/index.jsx#L18

```suggestion
        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p09kh14x.jpg',
```

Currently on the Russian service this image is `'https://ichef.bbci.co.uk/images/ic/512x512/p0776f5z.jpg'` which generates multiple image sources with the same dimensions.

<img width="957" alt="Screenshot 2021-08-18 at 13 38 51" src="https://user-images.githubusercontent.com/4798332/129899920-05f4b3be-2441-43ef-b80e-2f82d926c07f.png">

Reply:
Ooh nice catch - updated all these (+ russian) to use `$recipe`